.github/workflows/scheduled-list-update.yml

```diff
@@ -0,0 +1,45 @@
+name: Automate the update of stock list files
+
+on:
+  schedule:
+    # Asia/Kuala_Lumpur around 6pm, weekdays only
+    - cron: "0 10 * * 1,2,3,4,5"
+
+#on: [push]
+
+jobs:
+  build-and-commit:
+    name: Rebuild list files then commit them
+    runs-on: ubuntu-22.04
+
+    defaults:
+      run:
+        working-directory: packages/data
+
+    steps:
+      - name: Checkout the source
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.9.0
+
+      - name: Install NPM modules
+        uses: bahmutov/npm-install@HEAD
```

Comment:
is this using pnpm install? also we want to maintain the package version so that our ci/cd have exact lib version with our local